tests/iana_registries/bootp_dhcp_parameters.rflx

```diff
@@ -0,0 +1,1101 @@
+
+       -- Ref: rfc4578, alternative_name = Ethernet_Interface_Variable_Length_String, alte
+       -- rnative_name = Discrimination_String_To_Identify_Vendor
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_130 => 130,
+
+       -- Ref: rfc4578, alternative_name = Remote_Statistics_Server_Ip_Address_131
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_131 => 131,
+
+       -- Ref: rfc4578, alternative_name = Ieee_802_1q_Vlan_Id
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_132 => 132,
+
+       -- Ref: rfc4578, alternative_name = Ieee_802_1d_P_Layer_2_Priority
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_133 => 133,
+
+       -- Ref: rfc4578, alternative_name = Diffserv_Code_Point_Dscp_For_Voip_Signalling_An
+       -- d_Media_Streams
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_134 => 134,
+
+       -- Ref: rfc4578, alternative_name = Http_Proxy_For_Phone_Specific_Applications
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_135 => 135,
+
+       -- Ref: rfc5192
+       --
+       Option_Pana_Agent => 136,
+
+       -- Ref: rfc5223
+       --
+       Option_V4_Lost => 137,
+
+       -- length = N, description = CAPWAP Access Controller addresses, Ref: rfc5417
+       --
+       Option_Capwap_Ac_V4 => 138,
+
+       -- length = N, description = a series of suboptions, Ref: rfc5678
+       --
+       Option_Ipv4_Address_Mos => 139,
+
+       -- length = N, description = a series of suboptions, Ref: rfc5678
+       --
+       Option_Ipv4_Fqdn_Mos => 140,
+
+       -- length = N, description = List of domain names to search for SIP User Agent Conf
+       -- iguration , Ref: rfc6011
+       --
+       Sip_Ua_Configuration_Service_Domains => 141,
+
+       -- length = N, description = ANDSF IPv4 Address Option for DHCPv4, Ref: rfc6153
+       --
+       Option_Ipv4_Address_Andsf => 142,
+
+       -- length = N, description = This option provides a list of URIs for SZTP bootstrap
+       --  servers, Ref: rfc8572
+       --
+       Option_V4_Sztp_Redirect => 143,
+
+       -- length = 16, description = Geospatial Location with Uncertainty, Ref: rfc6225
+       --
+       Geoloc => 144,
+
+       -- length = 1, description = Forcerenew Nonce Capable, Ref: rfc6704
+       --
+       Forcerenew_Nonce_Capable_145 => 145,
+
+       -- length = N, description = Information for selecting RDNSS, Ref: rfc6731
+       --
+       Rdnss_Selection => 146,
+
+       -- length = N, description = The name of the peer DOTS agent., Ref: rfc8973
+       --
+       Option_V4_Dots_Ri => 147,
+
+       -- length = N (the minimal length is 4), description = N/4 IPv4 addresses of peer D
+       -- OTS agent(s)., Ref: rfc8973
+       --
+       Option_V4_Dots_Address => 148,
+
+       -- Ref: rfc5859, alternative_name = Etherboot, alternative_name = Grub_Configuratio
+       -- n_Path_Name
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_150 => 150,
```

Comment:
The `TFTP server address` is missing as "alternative_name":

```
150 | TFTP server address | &nbsp; | &nbsp; | [RFC5859]
150 | Etherboot | &nbsp; | &nbsp; | &nbsp;
150 | GRUB configuration path name | &nbsp;
```

Reply:
Now present in 5812afdd